src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss

```diff
@@ -2,26 +2,30 @@
 
 $block: '.#{variables.$ns}multiple-tooltip';
 
-.g-root_theme_dark #{$block},
-.g-root_theme_dark-hc #{$block} {
-    --multiple-tooltip-item-bg-color: #424147; // TODO: color variable will appear in uikit 5
+.g-root_theme_dark,
+.g-root_theme_dark-hc {
+    --multiple-tooltip-item-bg-color: var(--g-color-private-white-100-solid);
     --multiple-tooltip-item-active-bg-color: var(--g-color-base-float-heavy);
     --multiple-tooltip-backdrop-background: linear-gradient(
         90deg,
-        #313036 0%,
-        rgba(49, 48, 54, 0.3) 100%
+        var(--g-color-base-background) 50%,
+        transparent
     );
     --multiple-tooltip-backdrop-filter: blur(16px);
 }
 
+.g-root_theme_dark-hc {
+    --multiple-tooltip-item-bg-color: var(--g-color-private-white-150-solid);
+}
+
 .g-root_theme_light #{$block},
 .g-root_theme_light-hc #{$block} {
-    --multiple-tooltip-item-bg-color: #7a7a7a; // TODO: color variable will appear in uikit 5
+    --multiple-tooltip-item-bg-color: var(--g-color-private-black-550-solid);
```

Comment:
Should return `TODO`, using private color is outside rules

Reply:
Do you have a proper solution for these cases?

Reply:
It should be noted that a non-private variable for this case is expected in the next major uikit